etl/src/main/java/org/pih/hivmigration/etl/sql/Setup.groovy

```diff
@@ -28,6 +28,25 @@ class Setup extends SqlMigrator {
             DELIMITER ;
         ''')
 
+        executeMysql("Create function uuid_hash", '''
+            -- A deterministic way to produce something UUID-ish (doesn't meet the full spec, but good enough for OpenMRS)
```

Comment:
I implemented this so that when I was looking at the data for an infant in my local OpenMRS and I re-ran the migration and refreshed the page, the EMR wouldn't crash with "Patient does not exist" because of the patient UUID changing. This ensures the patient UUID will stay the same across builds.

Reply:
Any thoughts on how much it differs from the full spec?  Do we potentially want to pull it out and just use the UUID() method after we are finished testing?

Reply:
I agree @mogoodrich - it's a cool idea for testing, but I think we should be cautious about moving this to prod.

Reply:
OpenMRS is software that considers "163554AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" a UUID. When I say "not to spec" I am referring to parts of the UUID spec that few people have ever read. There is no concern about OpenMRS ever having problems with these UUIDs.

Reply:
I'm not particularly worried about OpenMRS having problems with them.  I'm more worried about the level of uniqueness they provide and the likelihood of inadvertently creating 2 rows with the same uuid (whether in the same or different tables)

Reply:
Oh sure, yeah. That's why I added the comment to be careful to give it unique input. We should definitely look for that in PRs.

Reply:
The "16354AAA..." is a hacky think to appease Andy and his concept management which I never liked... but anyway I'll take a look downstream once this get merged....